MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -32,6 +34,13 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 				self?.refreshControl?.endRefreshing()
 			}
 		}
+        viewModel?.onErrorStateChange = { [weak self] viewModel in
+            if viewModel.hasMessage {
+                self?.errorView?.show(message: viewModel.message!)
```

Comment:
You can avoid the force-unwrap with an `if let`:

```suggestion
        if let message = viewModel.message {
            errorView?.show(message: message)
```